src/synthesizer/parametric/morphology.py

```diff
@@ -182,3 +219,104 @@ def compute_density_grid(self, xx, yy, units=kpc):
                 "Only kpc and milliarcsecond (mas) units are supported "
                 "for morphologies."
             )
+
+
+class PointSource(MorphologyBase):
+
+    """
+    A class holding the Sersic2D profile. This is a wrapper around the
+    astropy.models.Sersic2D class.
+    """
+
+    def __init__(
+        self,
+        offset=np.array([0., 0.])*kpc,
+        cosmo=None,
+        redshift=None,
+    ):
+        """
+        Initialise the morphology.
+        
+        Arguments
+            offset (unyt)
+                The [x,y] offset.
```

Comment:
Offset from what?

Reply:
Updated doc string - offset from the centre of the image.